fe/fe-core/src/main/java/com/starrocks/alter/LakeTableRollupBuilder.java

```diff
@@ -98,8 +98,8 @@ public AlterJobV2 build() throws StarRocksException {
                         .collect(Collectors.toList());
                 List<Long> shadowTabletIds = GlobalStateMgr.getCurrentState().getStarOSAgent().createShards(
                         originTablets.size(),
-                        olapTable.getPartitionFilePathInfo(partitionId),
-                        olapTable.getPartitionFileCacheInfo(partitionId),
+                        olapTable.getPartitionFilePathInfo(physicalPartitionId),
```

Comment:
this may cause the new shadowTablets storage path differs from the original ones, consider before/after upgrading, will there be compatible issues?

Reply:
> shadowTablets storage path differs from the original

Upgrading may indeed cause path switching, but the logic will not be problematic. This is because during actual operation, it relies on the shard information returned by starmgr to locate the tablet and the change will only affect the save path when creating the tablet.

Reply:
yes, the tablet itself is not a concern. However, when doing the vacuum, it holds the assumption that the tablets in the same partition stored in the same root location, and hence use the first tablet's fs to perform the vacuum for all the tablets.

Reply:
> yes, the tablet itself is not a concern. However, when doing the vacuum, it holds the assumption that the tablets in the same partition stored in the same root location, and hence use the first tablet's fs to perform the vacuum for all the tablets.

Yes. However, the root location provided by starmgr is not the real physical path. When actually executing file deletion, the system will find the physical path of the corresponding tablet to perform the deletion, so the files can be correctly deleted in the end.